crates/graphql/src/schema/query_root.rs

```diff
@@ -16,7 +17,7 @@ use tables::{
     store_config_jsons, storefronts,
 };
 
-use super::prelude::*;
+use super::{objects::nft::NftActivity, prelude::*};
```

Comment:
can pull nft:NftActivity from the existing object export. leave this as just the prelude.